rfd/0144-client-tools-updates.md

```diff
@@ -426,20 +446,24 @@ rpc GetAutoUpdate(GetAutoUpdateRequest) returns (GetAutoUpdateResponse);
 message GetAutoUpdateRequest {}
 
 message GetAutoUpdateResponse {
- // key is clusterURI.
-  map<string, Version> clusters = 1;
+  repeated Version versions = 1;
 }
 
 message Version {
-  bool tools_auto_update = 1;
-  string tools_version = 2;
+  string root_cluster_uri = 1;
+  bool tools_auto_update = 2;
+  string tools_version = 3;
+  string min_tools_version = 4;
 }
 ```
 The update logic will resolve the version to install using the following 
 precedence:
-1. `TELEPORT_TOOLS_VERSION` env var (including `off` value).
-2. Version for a cluster from the app state.
-3. The latest version from `GetAutoUpdate`.
+1. `TELEPORT_TOOLS_VERSION` env var, if defined.
+2. Version for a cluster from the app state, if defined.
```

Comment:
It's going to be `tools_version` from that cluster I assume?

I'd express that as "`tools_version` from the cluster manually selected to manage updates, if selected".

Reply:
Correct, I shortened it too much.